internal/appconfig/config.go

```diff
@@ -88,8 +88,9 @@ type Config struct {
 }
 
 type Deploy struct {
-	ReleaseCommand string `toml:"release_command,omitempty" json:"release_command,omitempty"`
-	Strategy       string `toml:"strategy,omitempty" json:"strategy,omitempty"`
+	ReleaseCommand string  `toml:"release_command,omitempty" json:"release_command,omitempty"`
+	Strategy       string  `toml:"strategy,omitempty" json:"strategy,omitempty"`
+	MaxUnavailable float64 `toml:"max_unavailable,omitempty" json:"max_unavailable,omitempty"`
```

Comment:
don't you prefer to use `*float64` here so we can check way down if it is set or not reliably. In a similar way the `--max-unavailable` cli flag should take precedence over the config setting if set. Using `*float64` serves there too.

Reply:
Ignore all that, my only real concern is that --max-unavailable doesn't take precedence over appConfig.Deploy.MaxUnavailable.

Reply:
I wrote this using *float64 initially, and halfway through changed it to just float64. I can't remember for the life of me why I changed it though!

fixed :)